app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -44,8 +44,15 @@ angular.module('dashboard')
     };
 
     self.topicSelected = function(topic) {
-        $log.debug("meetingStatusCtrl: topicSelected " + JSON.stringify(topic));
-        MEETING.set('TOPIC', topic);
+        StorageSrv.set(KEY.TOPIC, topic);
+    };
+
+    self.isSelected = function(topic) {
+        var selected = StorageSrv.get(KEY.TOPIC);
+        if (topic instanceof Object && selected instanceof Object) {
+            return topic.topicGuid === selected.topicGuid;
```

Comment:
if neither object has a property 'topicGuid' this will evaluate as true. You probably want to add 
` "topicGuid" in topic &&`
condition to the 'if' as first item.